tools/src/wyvern/stdlib/support/AST.java

```diff
@@ -301,14 +304,61 @@ public IExpr parseExpression(String input, GenContext ctx) throws ParseException
         }
     }
 
+    public TypedAST parseGeneratedModule(String input) throws ParseException {
+        InterpreterState state = getLocalThreadInterpreter();
+        GenContext genContext = Globals.getGenContext(state);
+        try {
+            TypedAST typAST = TestUtil.getNewAST(input.trim() + "\n", "TSL Parse");
```

Comment:
Please change the second parameter of the getNewAST to something else ("Module gen", for example).